python/paddle/framework/random.py

```diff
@@ -42,10 +42,35 @@ def manual_seed(seed):
 
     seed = int(seed)
 
+    if core.is_compiled_with_cuda():
+        print("PYTHON>>>>>> gpu nums: {}".format(core.get_cuda_device_count()))
+        for i in range(core.get_cuda_device_count()):
+            core.default_cuda_generator(i)._is_init_py = True
+            core.default_cuda_generator(i).manual_seed(seed)
+
     core.default_cpu_generator()._is_init_py = True
     return core.default_cpu_generator().manual_seed(seed)
 
 
+def get_cuda_state():
```

Comment:
If it is a public API, please add some docs.

Reply:
Added